### PR TITLE
Update stateofthemap-eu.js

### PR DESCRIPTION
**Pull request by jbelien**
Update `stateofthemap.eu` DNS for State of the Map EU 2023 organized by @osmbe

The `stateofthemap.eu` website will be hosted using GitHub Pages.
You can find the list of IP addresses in GitHub documentation: https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain

We're not planning to use `www.stateofthemap.eu` (unless you request it).

We will setup a Fastmail account for `@stateofthemap.eu` email addresses. It's not configured yet so I will create a separate PR when we're ready with this.

(ping @joostschouppe)

### CODE REVIEW

**Comment by tomhughes** [comment]
You can't use more than one name with github pages anyway as there is no way to configure multiple names at the github end.

**Comment by gravitystorm** [comment]
From a non-technical user's point of view, having a www -> apex redirect is much more preferable than www -> NXDOMAIN.

**Comment by jbelien** [comment]
> You can't use more than one name with github pages anyway as there is no way to configure multiple names at the github end.

Indeed. As I said, this is not a problem, we only plan to use `https://stateofthemap.eu/`.

> From a non-technical user's point of view, having a www -> apex redirect is much more preferable than www -> NXDOMAIN.

For most of our OSMBE domain name, we have a server that we use for such redirections.
So we can easily setup a redirect from `https://www.stateofthemap.eu/` to `https://stateofthemap.eu/`.
Do you want me to already set that up and update this PR accordingly ?

**Review by Firefishy — APPROVED** [review]

**Comment by Firefishy** [comment]
What is the repo holding the github pages site? It will be good to add the CNAME for the `www` record.

**Comment by jbelien** [comment]
> What is the repo holding the github pages site?

https://github.com/osmbe/sotm-eu-2023

> It will be good to add the CNAME for the `www` record.

No worries, I'll take care of it ASAP.

**Comment by Firefishy** [comment]
@jbelien 
The CNAME addition should be:
```
  // Main web server and it's aliases

  A("@", "185.199.108.153"),
  A("@", "185.199.109.153"),
  A("@", "185.199.110.153"),
  A("@", "185.199.111.153"),
  
  CNAME("www", "osmbe.github.io.")
```

File: src/stateofthemap-eu.js [hunk 1]
```diff
@@ -15,11 +15,13 @@ D(DOMAIN, REGISTRAR, DnsProvider(PROVIDER),
...
+  // MX("@", 1, "mail.openstreetmap.at."),
 
   // Main web server and it's aliases
 
-  A("@", "88.198.206.107"),
-  A("www", "88.198.206.107")
+  A("@", "185.199.108.153"),
+  A("@", "185.199.109.153"),
+  A("@", "185.199.110.153"),
+  A("@", "185.199.111.153")
```
Comment (Firefishy) [inline src/stateofthemap-eu.js:28]: Missing www cname for github pages.

**Comment by tomhughes** [comment]
But as we know adding a CNAME like that will achieve nothing other than allowing spammers to insert a site at that name by creating a repository that publishes to github pages for it.

**Comment by Firefishy** [comment]
> But as we know adding a CNAME like that will achieve nothing other than allowing spammers to insert a site at that name by creating a repository that publishes to github pages for it.

`www` aliases are handled specially by github pages. It is the only supported subdomain which redirects to domain apex. eg: `www.stateofthemap.eu` -> `stateofthemap.eu`

**Comment by Firefishy** [comment]
The site URL in github pages needs to be fixed to resolve potential hijacking issue: https://github.com/osmbe/sotm-eu-2023/issues/1

**Comment by jbelien** [comment]
> @jbelien The CNAME addition should be:
> 
> ```
>   // Main web server and it's aliases
> 
>   A("@", "185.199.108.153"),
>   A("@", "185.199.109.153"),
>   A("@", "185.199.110.153"),
>   A("@", "185.199.111.153"),
>   
>   CNAME("www", "osmbe.github.io.")
> ```

Thanks, I've made the change. 👍

**Comment by jbelien** [comment]
> The site URL in github pages needs to be fixed to resolve potential hijacking issue: [osmbe/sotm-eu-2023#1](https://github.com/osmbe/sotm-eu-2023/issues/1)

Done!

**Review by Firefishy — APPROVED** [review]

**Comment by jbelien** [comment]
Thanks @Firefishy !

**Comment by Firefishy** [comment]
@jbelien Please do set enforce HTTPS. Also if the certificate is an issue, try follow the trouble shooting option of remove + save again. https://docs.github.com/en/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https#troubleshooting-certificate-provisioning-certificate-not-yet-created-error

**Comment by jbelien** [comment]
> @jbelien Please do set enforce HTTPS. Also if the certificate is an issue, try follow the trouble shooting option of remove + save again. https://docs.github.com/en/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https#troubleshooting-certificate-provisioning-certificate-not-yet-created-error

Already done! 👍

**Pull request merged**